src/api_routes_websocket.rs

```diff
@@ -194,7 +197,7 @@ async fn handle_messages(
           client_ip.clone(),
           connection_id,
           rate_limiter.get_ref(),
-          context.get_ref().clone(),
+          context.reset_request_count(),
```

Comment:
What's this one about? It resets the activitypub request counter after each successful websocket message?

Reply:
The counter is for each API call. In case of HTTP the actix-web middleware in the library generates a new `Data` struct with counter set to 0 for each incoming request. For websocket we need to do this manually. Otherwise the counter would keep increasing while the websocket connection is open, and after a couple of API requests every further Activitypub fetch would throw an error.